koap-core/src/commonTest/kotlin/MessageTest.kt

```diff
@@ -25,11 +25,52 @@ import com.juul.koap.Message.Code.Response.ServiceUnavailable
 import com.juul.koap.Message.Code.Response.Unauthorized
 import com.juul.koap.Message.Code.Response.UnsupportedContentFormat
 import com.juul.koap.Message.Code.Response.Valid
+import com.juul.koap.Message.Option.Accept
+import com.juul.koap.Message.Option.ContentFormat
+import com.juul.koap.Message.Option.ETag
+import com.juul.koap.Message.Option.Echo
+import com.juul.koap.Message.Option.IfMatch
+import com.juul.koap.Message.Option.IfNoneMatch
+import com.juul.koap.Message.Option.Observe
+import com.juul.koap.Message.Option.Observe.Registration.Deregister
+import com.juul.koap.Message.Option.Observe.Registration.Register
+import com.juul.koap.Message.Option.RequestTag
+import com.juul.koap.Message.Option.UriHost
+import com.juul.koap.Message.Option.UriPort
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class MessageTest {
 
+    @Test
+    fun optionToStringOverrides() {
+        // content format
```

Comment:
IMHO, these comments don't add any value.
If for organization, I think separate test functions would be a better alternative (e.g. `fun contentFormatOptionsToString() { .. }`).

```suggestion
```

Reply:
removed